ext/AvizExt/viz/clustering.jl

```diff
@@ -88,21 +90,21 @@ function ADRIA.viz.map(rs::Union{Domain,ResultSet}, data::AbstractArray,
 
     return f
 end
-function ADRIA.viz.map!(g::Union{GridLayout,GridPosition},
-    rs::Union{Domain,ResultSet}, data::AbstractArray, clusters::Vector{Int64};
-    opts::Dict=Dict(), axis_opts::Dict=Dict())
-
-    # Vector of summary statistics (default is mean) computed over timesteps
-    opts[:summary] = get(opts, :summary, mean)
-    data_stats = collect(ADRIA.metrics.per_loc(opts[:summary], data))
-
+function ADRIA.viz.map!(
+    g::Union{GridLayout,GridPosition},
+    rs::Union{Domain,ResultSet},
+    data::AbstractArray,

```

Comment:
```suggestion
    data::AbstractArray{<:Real},

```

Reply:
Docstrings needs to be updated too! We now expect user to provide the values for each location, so `data` is no longer an array, but always a vector